src/pages/markets/components/helper/_available-platforms.tsx

```diff
@@ -192,6 +195,14 @@ const AvailablePlatforms = ({
                         </StyledFlex>
                     </LocalizedLink>
                 )}
+                {deriv_ctrader && (
+                    <LocalizedLink to="/deriv-ctrader/">
+                        <StyledFlex direction="row" ai="center">
+                            <embed src={DerivcTrader} width="24" height="24" />
```

Comment:
Why embed tag? Can't we use image tag?

Reply:
hi bro,  i just followed the pattern in this file.